ic-stable-structures/src/structure/cache/btreemap.rs

```diff
@@ -1,26 +1,25 @@
-use std::cell::RefCell;
 use std::hash::Hash;
 
 use dfinity_stable_structures::{Memory, Storable};
-use mini_moka::unsync::{Cache, CacheBuilder};
+use mini_moka::sync::{Cache, CacheBuilder};
 
 use crate::structure::*;
 
 /// A LRU Cache for StableBTreeMap
 pub struct CachedStableBTreeMap<K, V, M>
 where
-    K: Storable + Clone + Hash + Eq + PartialEq + Ord,
-    V: Storable + Clone,
+    K: Storable + Clone + Send + Sync + 'static + Hash + Eq + PartialEq + Ord,
```

Comment:
Do we need to add `Send` and `Sync` constraints here and in the other places?
Since this is a library user may not need the collection to be `Send`.

Reply:
They are needed by the internal cache. Anyway, ic_stable_structure entries must be deserialized and this forecs the objects to be send/sync/static even without explicitly requiring it. So, from a user perspective, this changes nothing.
BTW, we need to remove moka, so this could change.